### DISCUSSION TITLE
Does live grep support file type filters?

### DISCUSSION THREAD

**Discussion by linrongbin16** [liuchengxu/vim-clap#970] [Q&A] [answered]
Does live grep support file type filters?

e.g. rg's `--iglob` option

**Comment by liuchengxu** [comment, marked as answer]
`live_grep` is unpolished in comparison to `grep` and I'm not motivated to improve `live_grep`, you can use `grep` and make use of the more friendly [search syntax](https://github.com/liuchengxu/vim-clap#search-syntax) to achieve the same goal.

**Reply by linrongbin16** [reply]
thanks
